26-register_publication/1625-creator_navigates_to_the_reference_tab_and_selects_reference_subtype_corrigendum.feature

```diff
@@ -0,0 +1,25 @@
+Feature: Creator selects Reference type Contribution to journal and selects subtype Corrigendum
+
+    Scenario: Creator can see fields related to Reference subtype "Corrigendum"
+        Given Creator begins registering a Publication in the Wizard
+        When they navigate to the Reference tab
+        And they select the Reference type "Contribution to journal"
+        And they select the Reference subtype "Corrigendum"
+        Then they see fields:
+            | Search box for "Journal article" |
+            | DOI                              |
+            | Volume                           |
+            | Issue                            |
+            | Pages from                       |
+            | Pages to                         |
+            | Article number                   |
+        And they see a disabled field for Journal based on selected Journal article
```

Comment:
hva betyr egentlig `based on selected Journal article`?

Reply:
når man velger Journal article (publikasjon) her: `Search box for "Journal article"`
så skal den valgte publikasjonen være knyttet til en tidsskrift, så det er den tidsskriften som skal vises her